### PR TITLE
blockdev: add page

### LINKED ISSUE #13699: Page request: blockdev
### Command description

Call block device ioctls from the command line

### Command details

_No response_

### Documentation

https://manned.org/man/blockdev

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

I saw it didn't have a tldr page. I have no idea how to use it so I didn't bother to create the page.

### PR DESCRIPTION
**Pull request by DarkMatter-999**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

Closes #13699

### CODE REVIEW

File: pages/linux/blockdev.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`blockdev --flushbufs /dev/sdX`
+
+- Get the physical block size:
+
+`blockdev --getpbsz /dev/sdX`
+
+- Set the read-ahead value to 128 sectors:
+
+`blockdev --setra 128 /dev/sdX`
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/linux/blockdev.md:36]: Do the device paths need to be absolute or can they be relative?

**Comment by DarkMatter-999** [comment]
Thank you very much,

Fixed the formatting ✌️

File: pages/linux/blockdev.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DarkMatter-999) [reply pages/linux/blockdev.md]: Yeah, the paths are required to be absolute.

File: pages/linux/blockdev.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`blockdev --flushbufs {{/dev/sdXY}}`
+
+- Get the physical block size:
+
+`blockdev --getpbsz {{/dev/sdXY}}`
+
+- Set the read-ahead value to 128 sectors:
+
+`blockdev --setra 128 {{/dev/sdXY}}`
```
Comment (Managor) [inline pages/linux/blockdev.md:36]: We aim to add sudo whenever superuser privileges are required

```suggestion
`sudo blockdev --report`

- Print a report for a specific device:

`sudo blockdev --report {{/dev/sdXY}}`

- Get the size of a device in 512-byte sectors:

`sudo blockdev --getsz {{/dev/sdXY}}`

- Set read-only:

`sudo blockdev --setro {{/dev/sdXY}}`

- Set read-write:

`sudo blockdev --setrw {{/dev/sdXY}}`

- Flush buffers:

`sudo blockdev --flushbufs {{/dev/sdXY}}`

- Get the physical block size:

`sudo blockdev --getpbsz {{/dev/sdXY}}`

- Set the read-ahead value to 128 sectors:

`sudo blockdev --setra 128 {{/dev/sdXY}}`
```

File: pages/linux/blockdev.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# blockdev
+
+> Call block device ioctls from the command line.
+> More information: <https://manned.org/man/blockdev>.
```
Comment (Managor) [inline pages/linux/blockdev.md:4]: Sorry I didn't notice this on the first pass. ```suggestion
> More information: <https://manned.org/blockdev>.
```

**Review by Managor — APPROVED** [review]
Nice. Looks good to me.

File: pages/linux/blockdev.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# blockdev
+
+> Call block device ioctls from the command line.
```
Comment (spageektti) [inline pages/linux/blockdev.md:3]:
```suggestion
> Manage, query, and manipulate block devices.
```

**Review by spageektti — APPROVED** [review]

**Comment by Managor** [comment]
Congrats on your first contribution.

**Pull request merged**